Sources/BrowserServicesKit/Common/Extensions/URLExtension.swift

```diff
@@ -137,6 +137,37 @@ extension URL {
         })
         return queryItem?.value
     }
+    
+    static func trimHostToETLD(host: String) -> String {
+        guard !host.isEmpty else {
+            return host
+        }
+        
+        var newHost = host
+        while newHost.contains(".") {
+            let comps = newHost.split(separator: ".").dropFirst()
+            if comps.count == 1 || comps.joined(separator: ".") == "co.uk" {
```

Comment:
This still looks like something to solve through TLD class :)

Reply:
I replaced the usage but forgot to remove my implementation 😅